testing/gerbv_example/example_board_al_custom/expected/back.ngc

```diff
@@ -414,9 +410,7 @@ X-4.81601 Y-2.92779 Z[#3+-0.04000]
 #2=[#516+[#517-#516]*0.35816]
 #3=[#1+[#2-#1]*0.99789]
 X-4.81377 Y-2.93518 Z[#3+-0.04000]
-#1=[#516+[#517-#516]*0.33331]
-#2=[#519+[#520-#519]*0.33331]
```

Comment:
Great!  I see here that we used to access a variable that is larger than `#517`, but in the new code, there is no usage of variables greater than `#517`.  Excellent!